.github/workflows/building-image.yml

```diff
@@ -0,0 +1,52 @@
+name: Building image
+
+on:
+  push:
+    branches: [ '**' ]
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+  build-all:
+    name: 'Building'
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ windows-2022 ]
+        node-version: [ 16.x ]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
```

Comment:
@acidicMercury8 I think this may be removed, we don't use npm here, and `node-version` should install node anyway?

Reply:
@PeyTy you're right, fixed